LayoutTests/svg/parser/whitespace-number-expected.txt

```diff
@@ -0,0 +1,1460 @@
+FAIL Test <number> trailing garbage, value: \r\n\t +17E-1a assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1.7000000476837158
+FAIL Test <number> trailing garbage, value: \r\n\t 17e+2a assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1700
+FAIL Test <number> trailing garbage, value: \r\n\t -47e assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got -47
+FAIL Test <number> trailing garbage, value: \r\n\t .1e assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 0.10000000149011612
+FAIL Test <number> trailing garbage, value: \r\n\t 0.35e assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 0.3499999940395355
+PASS Test <number> trailing garbage, value: \r\n\t 1e-10e
+FAIL Test <number> trailing garbage, value: \r\n\t +32e assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 32
+FAIL Test <number> trailing garbage, value: \r\n\t +17E-1e assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1.7000000476837158
+FAIL Test <number> trailing garbage, value: \r\n\t 17e+2e assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1700
+FAIL Test <number> trailing garbage, value: \r\n\t -47foo assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got -47
+FAIL Test <number> trailing garbage, value: \r\n\t .1foo assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 0.10000000149011612
+FAIL Test <number> trailing garbage, value: \r\n\t 0.35foo assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 0.3499999940395355
+PASS Test <number> trailing garbage, value: \r\n\t 1e-10foo
+FAIL Test <number> trailing garbage, value: \r\n\t +32foo assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 32
+FAIL Test <number> trailing garbage, value: \r\n\t +17E-1foo assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1.7000000476837158
+FAIL Test <number> trailing garbage, value: \r\n\t 17e+2foo assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1700
+FAIL Test <number> trailing garbage, value: \r\n\t -47)90 assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got -47
+FAIL Test <number> trailing garbage, value: \r\n\t .1)90 assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 0.10000000149011612
+FAIL Test <number> trailing garbage, value: \r\n\t 0.35)90 assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 0.3499999940395355
+PASS Test <number> trailing garbage, value: \r\n\t 1e-10)90
+FAIL Test <number> trailing garbage, value: \r\n\t +32)90 assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 32
+FAIL Test <number> trailing garbage, value: \r\n\t +17E-1)90 assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1.7000000476837158
+FAIL Test <number> trailing garbage, value: \r\n\t 17e+2)90 assert_approx_equals: expected 0 +/- 5.960464477539063e-8 but got 1700
```

Comment:
trailing garbage tests fail. I'm trying to understand if it's because the C++ normalize the code by dropping everything after parsing what is right. 

I also wonder what are the results in Firefox and Chrome for this test?

Reply:
I wonder if the tests should be separated in different categories. Just thinking.